KalturaWowzaServer/build.gradle

```diff
@@ -63,6 +63,7 @@ dependencies {
             "$WMSINSTALL_HOME/lib/joda-time-2.3.jar",
             "$WMSINSTALL_HOME/lib/commons-lang-2.6.jar",
             "$WMSINSTALL_HOME/lib/jid3lib-0.5.4.jar",
+            "$WMSINSTALL_HOME/lib/commons-lang3-3.4.jar"
```

Comment:
changed implementation and dropped the usage of Pair. No need for commons-lang3-3.4.jar.